featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -290,7 +450,17 @@ def pd_last(x):
 
 
 class Any(AggregationPrimitive):
-    """Test if any value is 'True'."""
+    """Test if any value is 'True'.
```

Comment:
should add in a list to make this more clear

Reply:
Fixed.